src/util/constants.ts

```diff
@@ -0,0 +1,22 @@
+export enum Colors {
+	White = 0xffffff,
+	Red = 0xff5c5c,
+	Orange = 0xf79454,
+	Yellow = 0xffdb5c,
+	Green = 0x5cff9d,
+	Blue = 0x5c6cff,
+	Pink = 0xb75cff,
+	Dark = 0x2f3136,
+	DiscordSuccess = 0x3ba55d,
+	DiscordDanger = 0xed4245,
+}
+
+export const DATE_FORMAT_LOGFILE = "YYYY-MM-DD_HH-mm-ss";
+export const DATE_FORMAT_WITH_SECONDS = "YYYY/MM/DD HH:mm:ss";
+
+export const DISCORD_USER_FLAG_SPAMMER = 1 << 20;
+
+export const TAB = " \u200B \u200B \u200B " as const;
```

Comment:
Are the spaces start and end here necessary?

Reply:
true, those are trimmed by discord